contracts/raidBasic.sol

```diff
@@ -35,14 +35,13 @@ contract RaidBasic is Initializable, Raid {
         participation[characterID] = true;
         // we drain ~12h of stamina from the character
         // we may want to have a specific lockout in the future
-        int128 traitMultiplier = 0;
         uint24 power = 0;
         totalPower += power;
         raiders.push(Raider(uint256(msg.sender), characterID, weaponID, power));
         emit RaiderJoined(msg.sender, characterID, weaponID, power);
     }
 
-    function completeRaid(uint256 seed) public override restricted {
+    function completeRaid(uint256 /*seed*/) public override restricted {
```

Comment:
@poshdan can we get rid of it altogether?

Reply:
Would love to. Though not something I'm looking forward to doing any time soon, seeing as it's already deployed, i dont know if there are any pitfalls to removing a contract.